src/components/Api.js

```diff
@@ -0,0 +1,185 @@
+export default class Api {
+  constructor(options) {
+    // constructor body
+    this.options = options;
+  }
+
+  getInitialCards() {
+    return fetch("https://around.nomoreparties.co/v1/cohort-3-en/cards/", {
```

Comment:
`https://around.nomoreparties.co/v1/cohort-3-en`

According to the task you need to pass full `headers` into the constructor of `Api`. See here https://disk.yandex.ru/i/J93dbJp7eypXKQ . And in the code you need to use only: `headers: this._headers` (not to duplicate the code)